unified-codes/apps/web/src/components/containers/detail/DetailProduct.tsx

```diff
@@ -64,8 +64,8 @@ export const DetailProductComponent: DetailProduct = ({ entity, fetchEntity }) =
         Product
       </Typography>
       <Typography className={classes.product} onClick={onProductClick}>
-        <Typography>{description}</Typography>
-        <Typography className={classes.code}>{code}</Typography>
+        {description}
```

Comment:
Got JSX validation error about nested `<p>` tags. I think this is going to be removed anyway soon

Reply:
No more DetailProduct.tsx